orchestrator/Makefile

```diff
@@ -6,6 +6,8 @@ install-tools:
 
 gen-mocks:
 	mockgen -destination=lib/coolfhir/client_mock.go -package=coolfhir -source=lib/coolfhir/client.go
+	mockgen -destination=careplancontributor/ehr/kafka_client_mock.go -package=ehr -source=careplancontributor/ehr/kafka_client.go
+	mockgen -destination=careplancontributor/ehr/nofifier_mock.go -package=ehr -source=careplancontributor/ehr/notifier.go
```

Comment:
this mock that was already there isn't actually used, we should probably remove it. We generate code with

```
go generate ./...
```

now. For a mock, this would be something like:

```
//go:generate mockgen -destination=./mock/fhirclient_mock.go -package=mock github.com/SanteonNL/go-fhir-client Client
package careplancontributor
```
(see `orchestrator/careplancontributor/service.go`).


All `go:generate` build instructions in the source tree are executed when `go generate ./...` is executed (from the app dir root). The pro is that we then don't need an external tool (make) to do stuff.

Reply:
Done.